resources/reproducing-ci.md

```diff
@@ -51,12 +51,14 @@ docker run \
   -it \
   --gpus all \
   --pull=always \
-  --network=host \
+  --env GITHUB_TOKEN=$(gh auth token) \
   --volume $PWD:/repo \
   --workdir /repo \
   rapidsai/ci-conda:cuda11.8.0-ubuntu22.04-py3.10
 ```
 
+Note: The `--env GITHUB_TOKEN=$(gh auth token)` flag is required to authenticate with GitHub for accessing artifacts. This token is automatically generated from your GitHub CLI authentication and passed to the container. Make sure you have the GitHub CLI (`gh`) installed and authenticated on your host machine.
```

Comment:
Does the container prompt for an interactive authentication if no token is provided?